R/Connect.R

```diff
@@ -82,6 +82,57 @@ createConnectionDetails <- function(dbms,
 }
 
 jdbcDrivers <- new.env()
+jdbcDrivers$defaultPath <- Sys.getenv("DATABASECONNECTOR_JAR_FOLDER", 
+                                      unset = path.expand("~/DatabaseConnectorDrivers"))
```

Comment:
Why store an environmental variable in a global variable? This way updates to the environmental variable will have no impact. Can't we just always refer to the environmental variable throughout the code?

Reply:
Yes we can. I want to provide a default path in the event that the environment variable is not set. I'll make the change

Reply:
I'm pretty sure CRAN will not allow us to default to a path in the user's folder, so I recommend the following: There are I think 3 places where the path to the jars must be specified:
1. downloadJdbcDrivers()
2. connect()
3. createConnectionDetails()

I recommend each of these just get a `pathToDriver` argument (the latter two already have that), that defaults to the environmental variable. So the user can choose to set the folder manually when calling the function, or setting it in the environmental variable. If no folder is specified, a meaningful error should be thrown.

Reply:
I think I've made the necessary changes according to your suggestions.